server/src/main/java/org/elasticsearch/common/logging/RateLimitingFilter.java

```diff
@@ -34,14 +36,15 @@
  * passed by a user on a HTTP header.
  * This filter works by using a lruKeyCache - a set of keys which prevents a second message with the same key to be logged.
  * The lruKeyCache has a size limited to 128, which when breached will remove the oldest entries.
+ * <p>
+ * It is possible to disable use of `x-opaque-id` as a key with {@link RateLimitingFilter#setUseXOpaqueIdEnabled(boolean) }
  *
- * It is possible to disable use of `x-opaque-id` as a key with {@link RateLimitingFilter#setUseXOpaqueId(boolean) }
  * @see <a href="https://logging.apache.org/log4j/2.x/manual/filters.htmlf">Log4j2 Filters</a>
  */
 @Plugin(name = "RateLimitingFilter", category = Node.CATEGORY, elementType = Filter.ELEMENT_TYPE)
 public class RateLimitingFilter extends AbstractFilter {
-
-    private volatile boolean useXOpaqueId = true;
+    // a flag to disable/enable use of xOpaqueId controlled by changing cluster setting
+    private volatile boolean useXOpaqueIdEnabled = true;
```

Comment:
Why the addition of "enabled"? The "use" prefix already denotes a boolean.

Reply:
true - it does not make sens to me now either. I can't remember what I was thinking when renaming..